hal/halx86/acpi/madt.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Source File for MADT parses 
+ * PROGRAMMERS: Copyright 2021 Justin Miller <justinmiller100@gmail.com>
```

Comment:
Please re-check the new headers format; I think "PROGRAMMERS" got replaced by "COPYRIGHT".